app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/ReserveView.java

```diff
@@ -39,8 +40,53 @@ public void setEndDateTextView(String date) {
 
     @Override
     public void returnToMainActivity() {
-        Intent intent = new Intent(getActivity(), MainActivity.class);
-        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-        getActivity().startActivity(intent);
+        Activity activity = getActivity();
+        if (activity != null)
+            getActivity().finish();
+    }
+
+    @Override
+    public int getParkingLot() {
+        String lot = binding.editTextNewReservationActivityLotNumber.getText().toString();
+        if (lot.isEmpty())
+            return - ConstantUtils.INT_ONE; //-1 used in model to check if user has set a parking lot
```

Comment:
create a MINUS_ONE_INT constant instead of this pls

Reply:
done